redfish-core/lib/processor.hpp

```diff
@@ -1159,6 +1184,110 @@ inline void handleProcessorCollectionHead(
         "</redfish/v1/JsonSchemas/ProcessorCollection/ProcessorCollection.json>; rel=describedby");
 }
 
+inline void
+    setProcessorData(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                     const dbus::utility::MapperServiceMap& serviceMap,
+                     const std::string& objectPath,
+                     std::optional<bool> locationIndicatorActive)
```

Comment:
This optional part must already been handled.
```
bool  locationIndicatorActive)
```

Reply:
Good catch! Fixed it.